src/consts.h

```diff
@@ -42,6 +42,7 @@
 #define   GRAVITY            32.2           // accel. of gravity in US units
 #define   SI_GRAVITY         9.81           // accel of gravity in SI units
 #define   MAXFILESIZE        2147483647L    // largest file size in bytes
+#define   MULTI_INFIL        1              // multi infiltration types allowed
```

Comment:
@aerispaha, is this needed for any particular reason?

Reply:
I don't think this will be necessary if we plan to support this new feature by default. This was originally a flag used to toggle the feature off/on

Reply:
@aerispaha, I think the null assumption here is that all projects will have this available. Compile switches are not going to be a part of the community version here :-)